### PR TITLE
TASK-41618: fixed password overwrite when importing csv file containing user data 

### PR DESCRIPTION
**Pull request by nidhalnidhal**
when importing a csv file containing users , the password is overwritten which causes many conflicts and must lead to additional operations in order to inform the user about the changes happening to his password. in order to get rid of this not recommended behavior dealing with user password ,i refactored importuser method in UserRestResourcesV1 to prevent password overwrite.

### CODE REVIEW

File: component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java [hunk 1]
```diff
@@ -1321,7 +1321,8 @@ private String importUser(UserImportResultEntity userImportResultEntity,
     boolean onboardUser = !userObject.isNull("onboardUser") && userObject.getString("onboardUser").equals("true");
 
     User existingUser = organizationService.getUserHandler().findUserByName(userName, UserStatus.ANY);
-    if (existingUser != null) {
+    if (existingUser != null && !userObject.isNull("password")) {
```
Comment (plamarque) [inline component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java]: IMO we could log inndebug mode here to tell when we skipped password  writing . it may be useful to separate updates from initial rights or for feneral troubleshooting

File: webapp/portlet/src/main/resources/locale/portlet/Portlets_en.properties [hunk 2]
```diff
@@ -316,6 +316,10 @@ SpaceSettings.button.moveAfter=Move after
 SpaceSettings.title.addApplication=Space application installer
 SpaceSettings.title.deleteApplicationConfirm=Delete application ?
 SpaceSettings.deleApplicationConfirm=Would you like to remove application {0} from space? (application data will be kept)
+SpaceSettings.application.members.title=Members
```
Comment (plamarque) [inline webapp/portlet/src/main/resources/locale/portlet/Portlets_en.properties]: seems unrelated

File: webapp/portlet/src/main/webapp/space-settings/components/SpaceSettingApplications.vue [hunk 3]
```diff
@@ -60,13 +60,27 @@ export default {
...
+            id: app.id(),
+            removable: app.removable,
+          }));
         });
     },
     openDetail() {
       this.$spaceService.getSpaceApplications(this.spaceId)
         .then(applications => {
-          this.applications = applications;
+          this.applications = applications.map(app => ({
```
Comment (plamarque) [inline webapp/portlet/src/main/webapp/space-settings/components/SpaceSettingApplications.vue]: unrelated fix

**Review by plamarque — COMMENTED** [review]
you seem to have extra code that leaked in this PR. Also consider improving logging for that case and adding unit tests

File: webapp/portlet/src/main/resources/locale/portlet/Portlets_en.properties (hunk 2 above)
Comment (nidhalnidhal) [reply webapp/portlet/src/main/resources/locale/portlet/Portlets_en.properties]: this was added to the PR in order to resolve a conflict with feature/maintenance branch

File: webapp/portlet/src/main/webapp/space-settings/components/SpaceSettingApplications.vue (hunk 3 above)
Comment (nidhalnidhal) [reply webapp/portlet/src/main/webapp/space-settings/components/SpaceSettingApplications.vue]: this was added to the PR in order to resolve a conflict with feature/maintenance branch

**Review by ahamdi — DISMISSED** [review]

**Review by ahamdi — DISMISSED** [review]

**Review by ahamdi — APPROVED** [review]
